easybuild/easyconfigs/p/psutil/psutil-7.0.0-GCCcore-14.2.0.eb

```
@@ -0,0 +1,23 @@
+easyblock = 'PythonBundle'
+
+name = 'psutil'
+version = '7.0.0'
+
+homepage = 'https://github.com/giampaolo/psutil'
+description = """A cross-platform process and system utilities module for Python"""
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+
+builddependencies = [('binutils', '2.42')]
+
+dependencies = [('Python', '3.13.1')]
+
+exts_list = [
+    (name, version, {
+        'source_urls': ['https://github.com/giampaolo/psutil/archive'],
+        'sources': ['%(name)s-%(version)s.tar.gz'],
```

```suggestion
        'sources': ['release-%(version)s.tar.gz'],
```

Full path is still `https://github.com/giampaolo/psutil/archive/release-7.0.0.tar.gz`, this got messed up by checksum injection I think...